simulator/src/main/java/com/hedera/block/simulator/grpc/impl/PublishStreamObserver.java

```diff
@@ -58,6 +63,9 @@ public PublishStreamObserver(
      */
     @Override
     public void onNext(PublishStreamResponse publishStreamResponse) {
+        if (lastKnownStatuses.size() == lastKnownStatusesCapacity) {
```

Comment:
Same as in the comment in the other observer.

Reply:
Very good point, thanks 👍